DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -17,8 +17,10 @@
 use Doctrine\Common\Cache\RedisCache;
 use Doctrine\Common\DataFixtures\Loader as DataFixturesLoader;
 use Doctrine\Common\EventSubscriber;
+use Doctrine\ODM\MongoDB\Configuration as MongoDBConfiguration;
```

Comment:
I'm nitpicking, but it's all mongodb here. We can specify ODM.
```suggestion
use Doctrine\ODM\MongoDB\Configuration as ODMConfiguration;
```